src/Domain/Tags.hpp

```diff
@@ -16,7 +16,27 @@
 #include "Domain/Element.hpp"
 #include "Domain/GridNormal.hpp"
 
+class DataVector;
+template <size_t Dim>
+class Element;
+template <size_t Dim>
+class Index;
+template <size_t Dim, typename TargetFrame>
+class ElementMap;
+namespace Frame {
+struct Logical;
+struct Inertial;
+}  // namespace Frame
+
 namespace Tags {
+/// \ingroup DataBoxTagsGroup
+/// \ingroup ComputationalDomainGroup
+/// The logical coordinates in the Element
+template <size_t Dim>
+struct LogicalCoordinates : db::DataBoxTag {
```

Comment:
Can this be a compute item based on the extents?

Reply:
Yep! Good point!